pkg/psmdb/backup/pbm.go

```diff
@@ -53,8 +53,11 @@ func NewPBM(c client.Client, cluster *api.PerconaServerMongoDB) (*PBM, error) {
 	if err != nil {
 		return nil, errors.Wrapf(err, "get pods list for replset %s", rs.Name)
 	}
-
-	scr, err := secret(c, cluster.Namespace, cluster.Spec.Secrets.Users)
+	usersSecretName := cluster.Spec.Secrets.Users
+	if ok, _ := cluster.VersionGreaterThanOrEqual("1.5.0"); ok {
```

Comment:
Please, resync with the main branch. Versions check was changed in recent commits to be aligned with the PXC operator. https://github.com/percona/percona-server-mongodb-operator/commit/67550fd6107165186f0eb7854325db2caea7fbd0#diff-26032a2e2bd4a7ece3ddd6f8070b12d1R471-R508